### DISCUSSION TITLE
Can the SNN converted from ANN by spikingjelly work in simulator, such as Brian2?

### DISCUSSION THREAD

**Discussion by xiaoxiaochaochao** [fangwei123456/SpikingFlow#195] [Q&A] [answered]
Sir, I would like to ask you whether the SNN converted from ANN by spikingjelly can work in simulator such as Brian2? The snn_toolbox(https://github.com/NeuromorphicProcessorProject/snn_toolbox) is also a tool that can convert ANN to SNN. After conversion, the SNN can work in simulator, such as Brian2 and spiNNker.

**Comment by fangwei123456** [comment, marked as answer]
Hi, SJ supports ANN2SNN (see tutorials for more details). But the SNN is built by pytorch. I think it can not run in Brian2 and spiNNker. Extra conversion for these frameworks is needed (e.g., the neuron is not same).
@DingJianhao , what is your opinion? Do you know the snn_toobox?

**Reply by DingJianhao** [reply]
I have learned about SNN Toolbox, actually the original algorithm in SpikingJelly.ann2snn is the same as in SNN-TB: MaxNorm. We make adaptation so that conversion can also work in resnet and other relu-based archs. In that case, it is far more difficult to deliver good simulation methods in Brain2 and others. But indeed it is appealing. Our team are working on the loihi interface. Please stay tuned
